src/WithDelayedHandlers.ts

```diff
@@ -0,0 +1,82 @@
+import { branch, compose, defaultProps, lifecycle, withHandlers, withPropsOnChange } from "recompose";
```

Comment:
Are there tests for this?

Reply:
Only indirectly through tests on the `InfiniteScroll` component itself. I can add separate tests for the HOC

Reply:
The HOC is now tested